src/App.css

```diff
@@ -0,0 +1,211 @@
+* {
+  font-family: "Roboto", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.four.wide.column {
+  box-sizing: border-box;
+  max-width: 50%;
+  display: inline-block;
+
+  /* flex-direction: row; */
+  height: 30px;
+  width: 100%;
+}
+
+.ui.cards > .card > .image {
+  height: 250px;
+  padding: 20px;
+  background: #fff;
+  margin: auto;
+}
+
```

Comment:
- [ ]  If you look to the space provided form the design, you will see that the list of products are very close and next to another. so I don't suggest you to bind product next to another but you can but a small border that surround your products so that the list of product can resemble to the one provided from the design.